src/screens/Proposals.js

```diff
@@ -261,7 +261,13 @@ const IdAndTitle = ({ id, name, selectProposal }) => {
   }, [id, selectProposal])
 
   return (
-    <Link onClick={handleOnClick}>
+    <Link
+      onClick={handleOnClick}
+      css={`
+      white-space: normal;
+      text-align: left;
+    `}
```

Comment:
Ah, could we fix the formatting? 

```suggestion
      css={`
        white-space: normal;
        text-align: left;
      `}
```

Reply:
Done! Sorry been a long day.

Reply:
ah this formatting stuff might be fixed when you save the changes to the file thanks to the .eslint configuration, if that is not working to you let us know